.gitignore

```diff
@@ -38,3 +38,4 @@ jspm_packages
 
 # Ignore package-lock
 package-lock.json
+.vscode
```

Comment:
Please ensure this file ends with a newline.

Reply:
done